hnn_core/tests/test_parallel_backends.py

```diff
@@ -182,7 +182,7 @@ def test_compare_hnn_core(self, run_hnn_core_fixture, backend, n_jobs=1):
                                      'L2_pyramidal': 113,
                                      'L5_pyramidal': 395,
                                      'L5_basket': 85,
-                                     'evdist1': 234,
```

Comment:
how come the test passed before?

Reply:
The other 35 spikes that correspond to L5 basket cells were silently ignored due to the fact that their connectivity has zero synaptic weight.